catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -310,6 +312,26 @@ private void removeAllClonedSpritesFromStage() {
 		StageActivity.resetNumberOfClonedSprites();
 	}
 
+	public List<Sprite> getAllClonesOfSprite(Sprite sprite) {
+		List<Sprite> clonesOfSprite = new ArrayList<Sprite>();
+		Pattern pattern = createCloneRegexPattern(sprite.getName());
+
+		for (Sprite spriteOfStage : sprites) {
+			Matcher matcher = pattern.matcher(spriteOfStage.getName());
```

Comment:
You could skip this if `spriteOfStage.isClone` is false

Reply:
Done :+1: